articles/handling-bias-in-machine-learning/index.md

```diff
@@ -0,0 +1,151 @@
+
+![hero](/engineering-education/articles/handling-bias-in-machine-learning/hero.jpg)
+
+[Image Source](https://images.unsplash.com/photo-1541844053589-346841d0b34c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60)
+
+We are increasingly dependent on machine learning in our daily lives. An example is in the every-day products we use. However, these algorithms are created by human beings. As a result, algorithms may contain the same biases humans possess. An algorithm displaying say, sexist, or racist bias has the potential to cause more harm than good. It is thus important for algorithms to make predictions that are accurate and fair.
+
+In this article, we cover:
+
+1. What bias is in machine learning
+
+2. Types of bias.
+
+3. Why you should care about bias in machine learning algorithms.
+
+4. Techniques to handle bias.
+
+### Prerequisites
+
+Basic understanding of machine learning concepts.
+
```

Comment:
You could link a few section.io articles over here so that beginners could read to understand the basics of ML. 
Check out the articles here: http://section.io/engineering-education